{"TimeStamp":"2025-01-23T21:33:32.756771921Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian systemd[1]: session-319.scope: Succeeded.\nJan 23 21:33:31 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 599.\nJan 23 21:33:31 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:31 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:31 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:31 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:31] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:31 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:31] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:32] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 35474: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 37156: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 35474: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:32] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:32.758459132Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:33:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:30 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:30 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:30 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:30 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.258.trace\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:30 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.258.trace\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:30 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/rm -rf /var/log/discern/data/bash-data.txt /var/log/discern/data/cpu-load-data.txt /var/log/discern/data/file-data.txt /var/log/discern/data/logs-data.txt /var/log/discern/data/network-data.txt /var/log/discern/data/proc-cpu-data.txt /var/log/discern/data/proc-mem-data.txt /var/log/discern/data/proc-new-data.txt\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:30 debian sshd[37174]: Received disconnect from 192.168.254.2 port 55230:11: disconnected by user\nJan 23 21:33:30 debian sshd[37174]: Disconnected from user blankcanvas 192.168.254.2 port 55230\nJan 23 21:33:30 debian sshd[37153]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:33:30 debian systemd-logind[388]: Session 319 logged out. Waiting for processes to exit.\nJan 23 21:33:30 debian systemd-logind[388]: Removed session 319.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:32.763275433Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:30] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:30 debian systemd[1]: session-319.scope: Succeeded.\nJan 23 21:33:31 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 599.\nJan 23 21:33:31 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:31 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:31 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:31 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:31] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:31 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:31] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:32] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 35474: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 37156: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:32] [Proc] error saving proc-mem info for pid 35474: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:32] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:33:32 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:32] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:32 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:32] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:37.779865991Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:33:34 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:34 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:34 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:34] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:34 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:34] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 37156\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 35899\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 36323\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 37157\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 37156: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 35899: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 36323: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 37157: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:33:35 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:35] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 601.\nJan 23 21:33:37 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:37 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:37 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 35474: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 37156: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:37] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:37.781974905Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:33:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:33:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 23 21:33:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nJan 23 21:33:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nJan 23 21:33:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nJan 23 21:33:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:37 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:37.790644801Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:33:34 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:34] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:34 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:34] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 37156\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 35899\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 36323\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] stripping info for: 37157\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 37156: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 35899: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 36323: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:35] [Proc] error saving proc-new info for pid 37157: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:35 debian discern-proc-sorcerer[35474]: #033[36mINFO#033[0m[21:33:35] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:33:35 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:35] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 601.\nJan 23 21:33:37 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:37 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:37 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 35474: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-proc-sorcerer[35474]: #033[31mERRO#033[0m[21:33:37] [Proc] error saving proc-cpu info for pid 37156: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:37] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:33:37 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:37] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:37 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:37] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:42.814949404Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:33:38 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:38] [Proc] error saving proc-mem info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:39 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:39] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:40 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:40] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:40 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:40] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:40 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 602.\nJan 23 21:33:40 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:40 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:40 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:41 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:41] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian systemd[1]: Started Session 322 of user blankcanvas.\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian systemd[1]: session-322.scope: Succeeded.\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian systemd[1]: Started Session 323 of user blankcanvas.\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:42] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:42.816887277Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.260.trace\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.260.trace\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sshd[37346]: Received disconnect from 192.168.254.2 port 41784:11: disconnected by user\nJan 23 21:33:42 debian sshd[37346]: Disconnected from user blankcanvas 192.168.254.2 port 41784\nJan 23 21:33:42 debian sshd[37340]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:33:42 debian systemd-logind[388]: Session 322 logged out. Waiting for processes to exit.\nJan 23 21:33:42 debian systemd-logind[388]: Removed session 322.\nJan 23 21:33:42 debian sshd[37381]: Accepted publickey for blankcanvas from 192.168.254.2 port 41796 ssh2: RSA SHA256:3I/WEXJMnChkvSdfEeoZ6YTUz2OrccqDruFNY41qR5o\nJan 23 21:33:42 debian sshd[37381]: pam_unix(sshd:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nJan 23 21:33:42 debian systemd-logind[388]: New session 323 of user blankcanvas.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:42.826370229Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:33:40 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:40] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:40 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:40] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:40 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 602.\nJan 23 21:33:40 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:40 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:40 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:41 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:41] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian systemd[1]: Started Session 322 of user blankcanvas.\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:33:42] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian systemd[1]: session-322.scope: Succeeded.\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian systemd[1]: Started Session 323 of user blankcanvas.\nJan 23 21:33:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:42] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:33:42 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:42] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:42 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:42] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:47.842007514Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:44 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 603.\nJan 23 21:33:44 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:44 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:44 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:44 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:44 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:45 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:46 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:46] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:46 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:46] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:47 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 604.\nJan 23 21:33:47 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:47 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:47 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:47 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:47] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:47.843909457Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.261.trace\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.261.trace\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:42 debian sshd[37387]: Received disconnect from 192.168.254.2 port 41796:11: disconnected by user\nJan 23 21:33:42 debian sshd[37387]: Disconnected from user blankcanvas 192.168.254.2 port 41796\nJan 23 21:33:42 debian sshd[37381]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:33:42 debian systemd-logind[388]: Session 323 logged out. Waiting for processes to exit.\nJan 23 21:33:42 debian systemd-logind[388]: Removed session 323.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:47.851698078Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:43 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:43] [Proc] error saving proc-mem info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:44 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 603.\nJan 23 21:33:44 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:44 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:44 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:44 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:44 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:44] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:45 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:46 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:46] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:46 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:46] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:47 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 604.\nJan 23 21:33:47 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:47 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:47 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:47 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:47] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:33:47 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:47] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:47 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:47] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:52.867800236Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:33:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:2d::d#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2801:1b8:10::b#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:503:ba3e::2:30#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:7fe::53#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:12::d0d#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:dc3::35#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:1::53#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:9f::42#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:7fd::1#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:2::c#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:2f::f#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:503:c27::2:30#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:a8::e#53\nJan 23 21:33:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:52 debian named[37720]: resolver priming query complete\nJan 23 21:33:52 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:52] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] saving config specified in env\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] saving config and playbooks from home\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] saving config from ~ and /etc\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] finished reading. looping\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] Saving playbooks from /etc/ansible\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[31mERRO#033[0m[21:33:52] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nJan 23 21:33:52 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:52] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:52.869731979Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:33:52 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:52 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:52 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:52 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:52 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.267.trace\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:52 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.267.trace\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:52 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/systemctl restart named\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:52 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:52 debian sshd[37686]: Received disconnect from 192.168.254.2 port 41850:11: disconnected by user\nJan 23 21:33:52 debian sshd[37686]: Disconnected from user blankcanvas 192.168.254.2 port 41850\nJan 23 21:33:52 debian sshd[37680]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:33:52 debian systemd-logind[388]: Session 329 logged out. Waiting for processes to exit.\nJan 23 21:33:52 debian systemd-logind[388]: Removed session 329.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:52.876654475Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2801:1b8:10::b#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:503:ba3e::2:30#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:7fe::53#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:12::d0d#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:dc3::35#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:1::53#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:9f::42#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:7fd::1#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:2::c#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:2f::f#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:503:c27::2:30#53\nJan 23 21:33:52 debian named[37720]: network unreachable resolving './NS/IN': 2001:500:a8::e#53\nJan 23 21:33:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:52 debian named[37720]: resolver priming query complete\nJan 23 21:33:52 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:52] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] saving config specified in env\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] saving config and playbooks from home\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] saving config from ~ and /etc\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] finished reading. looping\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:33:52] [Ansible] Saving playbooks from /etc/ansible\nJan 23 21:33:52 debian discern-ansible-sorcerer[2673]: #033[31mERRO#033[0m[21:33:52] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nJan 23 21:33:52 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:52] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:33:52 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:52] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:52 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:52] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:57.891328275Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:7fd::1#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:dc3::35#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:2::c#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:503:ba3e::2:30#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:2f::f#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:1::53#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:2d::d#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:9f::42#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:503:c27::2:30#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:12::d0d#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:a8::e#53\nJan 23 21:33:55 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:55] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:55 debian named[37834]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 20326 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:33:55 debian named[37834]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 38696 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:33:56 debian named[37834]: timed out resolving './DNSKEY/IN': 10.1.2.3#53\nJan 23 21:33:56 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:503:c27::2:30#53\nJan 23 21:33:56 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:56] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:56 debian named[37834]: resolver priming query complete\nJan 23 21:33:56 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 607.\nJan 23 21:33:56 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:56 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:56 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:57] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:57.893254197Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:33:54 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:54 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.269.trace\nJan 23 21:33:54 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:54 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:54 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.269.trace\nJan 23 21:33:54 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:54 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:54 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/systemctl restart named\nJan 23 21:33:54 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:33:54 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:54 debian sshd[37795]: Received disconnect from 192.168.254.2 port 41566:11: disconnected by user\nJan 23 21:33:54 debian sshd[37795]: Disconnected from user blankcanvas 192.168.254.2 port 41566\nJan 23 21:33:54 debian sshd[37789]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:33:54 debian systemd-logind[388]: Session 331 logged out. Waiting for processes to exit.\nJan 23 21:33:54 debian systemd-logind[388]: Removed session 331.\nJan 23 21:33:56 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:33:56 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:33:56 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:33:56 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:56 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:56 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:33:56 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:56 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:33:56 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:33:57.900717206Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:2::c#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:503:ba3e::2:30#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:2f::f#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:1::53#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:2d::d#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:9f::42#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:503:c27::2:30#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:12::d0d#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nJan 23 21:33:55 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:500:a8::e#53\nJan 23 21:33:55 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:55] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:55 debian named[37834]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 20326 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:33:55 debian named[37834]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 38696 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:33:56 debian named[37834]: timed out resolving './DNSKEY/IN': 10.1.2.3#53\nJan 23 21:33:56 debian named[37834]: network unreachable resolving './DNSKEY/IN': 2001:503:c27::2:30#53\nJan 23 21:33:56 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:33:56] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:56 debian named[37834]: resolver priming query complete\nJan 23 21:33:56 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 607.\nJan 23 21:33:56 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:56 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:33:56 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:57] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:57] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:57] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:02.917477146Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:57] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:33:57] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:57] [Logs] Sending has completed\nJan 23 21:33:58 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:33:58] [Proc] stripping info for: 37834\nJan 23 21:33:58 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:58] [Proc] error saving proc-new info for pid 37834: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:58 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:33:58] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:33:58 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:58] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 37834: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 37834: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:00 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 608.\nJan 23 21:34:00 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:00 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:00 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:00 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:34:00] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:02 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:02] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:02.926020172Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:33:57 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:33:57] [Logs] Sending has completed\nJan 23 21:33:58 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:33:58] [Proc] stripping info for: 37834\nJan 23 21:33:58 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:58] [Proc] error saving proc-new info for pid 37834: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:58 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:33:58] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:33:58 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:33:58] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 37834: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:33:59 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:33:59] [Proc] error saving proc-mem info for pid 37834: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:00 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 608.\nJan 23 21:34:00 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:00 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:00 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:00 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:34:00] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:02 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:02] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:34:02 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:34:02] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:07.940646664Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:05 debian named[37834]: timed out resolving 'com/DS/IN': 10.1.2.3#53\nJan 23 21:34:05 debian named[37834]: network unreachable resolving 'com/DS/IN': 2001:7fd::1#53\nJan 23 21:34:05 debian named[37834]: network unreachable resolving 'com/DS/IN': 2001:dc3::35#53\nJan 23 21:34:06 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 610.\nJan 23 21:34:06 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:06 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:06 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:07 debian named[37834]: timed out resolving 'google.com/DS/IN': 10.1.2.3#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:dc3::35#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:502:1ca1::30#53\nJan 23 21:34:07 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:34:07] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:501:b1f9::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:eea3::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:83eb::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:39c1::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:500:856e::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:231d::2:30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:a83e::2:30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:502:8cc::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:502:7094::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:d2d::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:500:d937::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:d414::30#53\nJan 23 21:34:07 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:07] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:07.942610356Z", "Location":"/var/log/auth.log", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:07.950666499Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:05 debian named[37834]: network unreachable resolving 'com/DS/IN': 2001:dc3::35#53\nJan 23 21:34:06 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 610.\nJan 23 21:34:06 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:06 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:06 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:07 debian named[37834]: timed out resolving 'google.com/DS/IN': 10.1.2.3#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:dc3::35#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:502:1ca1::30#53\nJan 23 21:34:07 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:34:07] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:501:b1f9::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:eea3::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:83eb::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:39c1::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:500:856e::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:231d::2:30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:a83e::2:30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:502:8cc::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:502:7094::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:d2d::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:500:d937::30#53\nJan 23 21:34:07 debian named[37834]: network unreachable resolving 'google.com/DS/IN': 2001:503:d414::30#53\nJan 23 21:34:07 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:07] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:34:07 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:34:07] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:07 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:34:07] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:12.966777467Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:09 debian named[37834]: no longer listening on 172.30.0.13#53\nJan 23 21:34:09 debian named[37834]: no longer listening on 10.1.1.3#53\nJan 23 21:34:09 debian named[37834]: no longer listening on 10.1.2.2#53\nJan 23 21:34:09 debian named[37834]: no longer listening on 10.0.3.2#53\nJan 23 21:34:09 debian named[37834]: no longer listening on ::1#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::8816:fbff:fe04:ad4a%2#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::a8c1:49ff:fe90:b0ca%3#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::400f:d4ff:fe47:d4cd%4#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::9419:78ff:fe6b:82f9%5#53\nJan 23 21:34:09 debian named[37834]: shutting down: flushing changes\nJan 23 21:34:09 debian named[37834]: stopping command channel on 127.0.0.1#953\nJan 23 21:34:09 debian named[37834]: stopping command channel on ::1#953\nJan 23 21:34:09 debian named[37834]: exiting\nJan 23 21:34:09 debian systemd[1]: named.service: Succeeded.\nJan 23 21:34:09 debian systemd[1]: Stopped BIND Domain Name Server.\nJan 23 21:34:09 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:34:09] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:09 debian systemd[1]: session-332.scope: Succeeded.\nJan 23 21:34:09 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:34:09] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:09 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 611.\nJan 23 21:34:09 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:09 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:09 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:10 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:34:10] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:12 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:12] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:12.968739200Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nJan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nJan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:34:09 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:09 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:12.975682065Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:09 debian named[37834]: no longer listening on 10.1.2.2#53\nJan 23 21:34:09 debian named[37834]: no longer listening on 10.0.3.2#53\nJan 23 21:34:09 debian named[37834]: no longer listening on ::1#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::8816:fbff:fe04:ad4a%2#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::a8c1:49ff:fe90:b0ca%3#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::400f:d4ff:fe47:d4cd%4#53\nJan 23 21:34:09 debian named[37834]: no longer listening on fe80::9419:78ff:fe6b:82f9%5#53\nJan 23 21:34:09 debian named[37834]: shutting down: flushing changes\nJan 23 21:34:09 debian named[37834]: stopping command channel on 127.0.0.1#953\nJan 23 21:34:09 debian named[37834]: stopping command channel on ::1#953\nJan 23 21:34:09 debian named[37834]: exiting\nJan 23 21:34:09 debian systemd[1]: named.service: Succeeded.\nJan 23 21:34:09 debian systemd[1]: Stopped BIND Domain Name Server.\nJan 23 21:34:09 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:34:09] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:09 debian systemd[1]: session-332.scope: Succeeded.\nJan 23 21:34:09 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:34:09] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:09 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 611.\nJan 23 21:34:09 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:09 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:09 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:10 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:34:10] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:12 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:12] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:34:12 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:34:12] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:12 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:34:12] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:17.992754590Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:13 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:13 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:13] [Proc] stripping info for: 37931\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:13] [Proc] stripping info for: 37932\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:13] [Proc] error saving proc-new info for pid 37931: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:13] [Proc] error saving proc-new info for pid 37932: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:13] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 37931: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:16 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:34:16] [Network] saved network slice\nJan 23 21:34:16 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 613.\nJan 23 21:34:16 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:16 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:16 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:16 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:34:16] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:17 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:17] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:18.012137788Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:13 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:13 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:13] [Proc] stripping info for: 37931\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:13] [Proc] stripping info for: 37932\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:13] [Proc] error saving proc-new info for pid 37931: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:13] [Proc] error saving proc-new info for pid 37932: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:13 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:13] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 37931: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:14 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:14] [Proc] error saving proc-mem info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:34:16 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:34:16] [Network] saved network slice\nJan 23 21:34:16 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 613.\nJan 23 21:34:16 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:16 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:16 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:16 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:34:16] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:17 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:17] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:23.035978865Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:19 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nJan 23 21:34:19 debian systemd[1]: run-user-1001.mount: Succeeded.\nJan 23 21:34:19 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:19 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nJan 23 21:34:19 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 23 21:34:19 debian systemd[1]: Removed slice User Slice of UID 1001.\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-0.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-1.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-10.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-100.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-101.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 615.\nJan 23 21:34:22 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:22 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:22 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-102.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-103.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-104.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-105.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-106.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-107.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-108.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:23 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:23] [Bash] Error saving cache-blankcanvas-109.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:23 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:23] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:23.043685766Z", "Location":"/var/log/auth.log", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:23.054709198Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:19 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nJan 23 21:34:19 debian systemd[1]: run-user-1001.mount: Succeeded.\nJan 23 21:34:19 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:19 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nJan 23 21:34:19 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 23 21:34:19 debian systemd[1]: Removed slice User Slice of UID 1001.\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-0.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-1.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-10.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-100.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-101.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 615.\nJan 23 21:34:22 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:22 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:22 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-102.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-103.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-104.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-105.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-106.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-107.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:22 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:22] [Bash] Error saving cache-blankcanvas-108.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:23 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:23] [Bash] Error saving cache-blankcanvas-109.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:23 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:23] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:28.077261759Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-26.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-27.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-28.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-29.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-3.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-30.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:28] [Bash] Error saving cache-blankcanvas-31.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:28] [Bash] Error saving cache-blankcanvas-32.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:28] [Bash] Error saving cache-blankcanvas-33.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:28] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:28.081578667Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:34:26 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmmsg.bt\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:26 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nJan 23 21:34:26 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmsg.bt\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:26 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:34:26 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:34:26 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:28.088330551Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-26.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-27.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:27] [Bash] Sent message safely to the Fusion Core\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-28.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-29.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-3.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:27 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:27] [Bash] Error saving cache-blankcanvas-30.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:28] [Bash] Error saving cache-blankcanvas-31.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:28] [Bash] Error saving cache-blankcanvas-32.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:28] [Bash] Error saving cache-blankcanvas-33.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:28 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:28] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:33.108643726Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-94.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-95.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-96.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-97.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-98.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-99.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-root-0.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-root-1.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:29] [Bash] finished sending logs. pausing...\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:32 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 618.\nJan 23 21:34:32 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:32 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:32 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:33] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:33.129358062Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-94.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-95.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-96.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-97.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-98.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-blankcanvas-99.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-root-0.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:34:29] [Bash] Error saving cache-root-1.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:34:29 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:34:29] [Bash] finished sending logs. pausing...\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:29 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:29] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:32 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 618.\nJan 23 21:34:32 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:32 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:32 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:33] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:38.155230179Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:33 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:33] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:34:33 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:34:33] [Network] saved network slice\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:35 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 619.\nJan 23 21:34:35 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:35 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:35 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:38] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:38.172468872Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:33 debian discern-proc-sorcerer[37314]: #033[36mINFO#033[0m[21:34:33] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:34:33 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:34:33] [Network] saved network slice\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:34 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:34] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:35 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 619.\nJan 23 21:34:35 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:35 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:35 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:38] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:43.195182248Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:34:39 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:39 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:39 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:42 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 621.\nJan 23 21:34:42 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:42 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:42 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] saving config specified in env\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] saving config and playbooks from home\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] saving config from ~ and /etc\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] finished reading. looping\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] Saving playbooks from /etc/ansible\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[31mERRO#033[0m[21:34:42] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nJan 23 21:34:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:43] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:34:43.209323621Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:34:39 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:39 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:39 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-cpu info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:40 debian discern-proc-sorcerer[37314]: #033[31mERRO#033[0m[21:34:40] [Proc] error saving proc-mem info for pid 37314: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:34:42 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 621.\nJan 23 21:34:42 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:42 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:34:42 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] saving config specified in env\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] saving config and playbooks from home\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] saving config from ~ and /etc\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] finished reading. looping\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:34:42] [Ansible] Saving playbooks from /etc/ansible\nJan 23 21:34:42 debian discern-ansible-sorcerer[2673]: #033[31mERRO#033[0m[21:34:42] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nJan 23 21:34:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:34:43] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
